bunsen/pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>com.google.fhir.analytics</groupId>
     <artifactId>root</artifactId>
-    <version>0.1.0-SNAPSHOT</version>
+    <version>0.2.7-SNAPSHOT</version>
```

Comment:
Just a suggestion to see if we can optimise the version to be present at root level as a property and all the modules inherit from this global property.

Reply:
So just to be clear, this `version` here references the version of the `root` which I think is required to define the `parent`. All modules under `pipelines` use this same version the way you have suggested. But for `bunsen` (and modules under it) I have kept the version as before (e.g., `0.5.14-SNAPSHOT` on line 14 below) since we have forked this and it had its own versioning.